VideoLocker/src/main/java/org/edx/mobile/util/IOUtils.java

```diff
@@ -0,0 +1,21 @@
+package org.edx.mobile.util;
+
+import android.support.annotation.NonNull;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.nio.charset.Charset;
+
+import okio.Okio;
+
+public class IOUtils {
+    @NonNull
+    public static String toString(@NonNull InputStream in, @NonNull Charset charset) throws IOException {
+        return Okio.buffer(Okio.source(in)).readString(charset);
```

Comment:
>  `Charset.defaultCharset()` always returns the `UTF-8` charset

Is that documented somewhere? I don't think that's true. It returns the client's default charset, which is *probably* UTF-8 but doesn't have to be.